Controller Verify SSL Breaks Availability Check with Valid Certs

### Issue
Check availability fails when controller URL is defined with an alternative port.

### Steps to Reproduce

1. Define:
```
PINAKES_CONTROLLER_URL=https://controller.example.org:443
PINAKES_CONTROLLER_VERIFY_SSL=true
```

2. Verify certificate on controller a good from the pinakes/catalog host:
```
curl https://controller.example.org:443 &> /dev/null && echo $?
0
```

### Expected Results
Platform in UI is available

### Observed Results
```
Mar 24 22:16:05 catalog.example.org bash[86689]: [2022-03-24 22:16:05,385] [87063] [ERROR] check_source_availability 140236529504512 Check availability failed on https://controller.example.org:443: expected str, bytes or os.PathLike object, not int
Mar 24 22:16:05 catalog.example.org bash[86689]: [2022-03-24 22:16:05,387] [87063] [ERROR] tasks 140236529504512 Source Automation Controller[https://controller.example.org:443] is unavailable, cannot refresh it
```

--- a/pinakes/main/inventory/task_utils/tower_api.py
+++ b/pinakes/main/inventory/task_utils/tower_api.py
@@ -3,6 +3,7 @@
     2. Get multiple objects with pagination info
     3. POST a Job Template/Workflow
 """
+from pathlib import Path
 from distutils.util import strtobool
 from django.conf import settings
 import requests
@@ -27,7 +28,10 @@ class TowerAPI:
         if verify_ssl is None:
             verify_ssl = settings.CONTROLLER_VERIFY_SSL
 
-        self.verify_ssl = strtobool(verify_ssl)
+        if Path(verify_ssl).is_file():
+            self.verify_ssl = verify_ssl
+        else:
+            self.verify_ssl = bool(strtobool(verify_ssl))
 
         self.headers = {"Authorization": f"Bearer {token}"}
 

--- a/pinakes/main/inventory/tests/task_utils/test_tower_api.py
+++ b/pinakes/main/inventory/tests/task_utils/test_tower_api.py
@@ -189,3 +189,22 @@ class TestTowerAPI:
         )
         with pytest.raises(RuntimeError, match=r"not found"):
             tower_api.post("/api/v2/job_templates/1/launch/", {"a": 1}, [])
+
+    testdata = [("y", True), ("n", False), ("true", True), ("false", False)]
+
+    @pytest.mark.parametrize("value,expected", testdata)
+    def test_verify_ssl_boolean(self, value, expected):
+        """Check if verify_ssl is proper boolean"""
+        tower_api = TowerAPI("https://www.example.com", "gobbledegook", value)
+        assert (tower_api.verify_ssl) == expected
+
+    def test_verify_ssl_ca_file(self, tmpdir):
+        """Check if verify_ssl is ca_file"""
+        ca_file = tmpdir.join("ca.crt")
+        ca_file.write("dummy")
+        tower_api = TowerAPI(
+            "https://www.example.com",
+            "gobbledegook",
+            str(ca_file),
+        )
+        assert (tower_api.verify_ssl) == str(ca_file)

